nixos/ec2/autoswap.nix

```diff
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.ec2.setupSwapFile;
+
+  inherit (lib) mkOption types;
+
+in {
+
+  options = {
+    ec2.setupSwapFile = {
```

Comment:
No, I'll change it. I think named like this because I hard-coded `/disk0` before.